README

```diff
@@ -64,7 +64,7 @@ Much, much more information is also available in the Open MPI FAQ:
 ===========================================================================
 
 The following abbreviated list of release notes applies to this code
-base as of this writing (March 2017):
+base as of this writing (February 2019):
```

Comment:
sad trombone :)

Reply:
Can fix momentarily.  Will just have to wait for CI.